code/modules/mob/living/carbon/human/species.dm

```diff
@@ -716,6 +716,67 @@
 	has_internals = 0
 	gear = list()
 
+/datum/species/zombie/necromorph
+	name = "Necromorph"
+	name_plural = "Necromorphs"
+	blood_color = "#1e0100"
+	slowdown = -1
+	brute_mod = 0.7
+	burn_mod = 0.9 //don't murderize me lasers
+	icobase = 'icons/mob/human_races/r_necro.dmi'
+	deform = 'icons/mob/human_races/r_necro.dmi'
+
+/datum/species/zombie/necromorph/handle_death(var/mob/living/carbon/human/H, gibbed)
+//	if(H && H.loc && H.stat == DEAD)
+	H.gib()
+
+/datum/species/zombie/necromorph/handle_post_spawn(var/mob/living/carbon/human/H)
+	if(H.hud_used)
+		qdel(H.hud_used)
+		H.hud_used = null
+//		H.create_mob_hud()
+		if(H.hud_used)
```

Comment:
This `if` can never be true as the code goes. Do you know what's going on here to keep it?

Reply:
Nope, I don't really get what is going on there. It seems that it wants to delete huds humans are using i suppose? But the masks tend to get deleted, and huds are kind of tied to uh... glasses and such?